src/Compilers/CSharp/Portable/BoundTree/BoundNode.cs

```diff
@@ -20,14 +20,20 @@ internal abstract partial class BoundNode
         [Flags()]
         private enum BoundNodeAttributes : byte
         {
-            HasErrors = 1 << 0,
-            CompilerGenerated = 1 << 1,
+            HasErrors = 0b0001,
```

Comment:
>0b0001 [](start = 24, length = 6)

nit: I find the `x << y` more readable. Then for masks, `Flag1 | Flag2 | ...`. In the literal format, it's hard to see the alignment of positions.

Reply:
See, I find the literal format more readable. I'll revert this to match the existing style, but that's why I changed it.

---
In reply to: [251670011](https://github.com/dotnet/roslyn/pull/32899#discussion_r251670011) [](ancestors = 251670011)